packages/dynamite/dynamite/pubspec.yaml

```
@@ -1,5 +1,13 @@
 name: dynamite
-version: 1.0.0
+version: 0.1.0
```

```suggestion
version: 0.1.0-dev.0
```

Needed so melos version will "upgrade" it to 0.1.0